stories/exercises/06_exercise.mdx

```diff
@@ -0,0 +1,62 @@
+# 06. Customize components using settings.
+
+In the last exercise, we familiarized ourselves with the USWDS theme settings. In this section, we'll take that a step further by using component specific settings.
```

Comment:
Throughout this exercise, I use guidance verbiage such as:

> we familiarized...
> If we inspect...
> we can change which token...

How do you feel about this @mejiaj ? Should I avoid the collective we when referring to the user?

Reply:
I don't see any harm in it. We can re-visit as this grows. 😄